src/mpid/ch4/netmod/ofi/ofi_am_impl.h

```diff
@@ -179,7 +185,8 @@ static inline int MPIDI_OFI_progress_do_queue(int vni_idx)
             (MPIDI_Global.cq_buffered_static_head + 1) % MPIDI_OFI_NUM_CQ_BUFFERED;
     }
 
-    if ((cq_entry.flags & FI_RECV) && (cq_entry.flags & FI_MULTI_RECV)) {
+  multi_recv:
+    if (cq_entry.flags & FI_MULTI_RECV) {
```

Comment:
Minor thing, but I think this can be
```C
    if (cq_entry.flags & FI_MULTI_RECV) {
multi_recv:
```
in which we can skip one comparison.

Reply:
@hajimefu Yes, this is nice point